AUTHOR

```diff
@@ -0,0 +1,16 @@
+Koeun Choi <koeun.choi at samsung dot com>
```

Comment:
Can we just put
Samsung Electronics Co., Ltd.
like google is doing in projects:
https://github.com/google/amber/blob/master/AUTHORS